wagtail/users/static_src/wagtailusers/scss/groups_edit.scss

```
@@ -16,6 +16,14 @@
         // stylelint-disable-next-line declaration-no-important
         font-size: 2.5em !important;
     }
+
+    .custom-permissions {
+        padding-bottom: 0;
```

The default bold feels a bit off here, maybe let's  make that look the same as other content.

```suggestion
        padding-bottom: 0;
        
        label {
            font-weight: inherit;
        }
```